packages/grant-explorer/src/features/discovery/CardBanner.tsx

```diff
@@ -87,11 +87,11 @@ export function RoundBanner(props: { roundId: string }) {
   );
 }
 
-export function CollectionBanner({ projects }: { projects: { id: string }[] }) {
+export function CollectionBanner({ projects }: { projects: string[] }) {
```

Comment:
If these are project ids or project names, rather than project objects, I'd call the variable `projectIds` or `projectNames`.